src/components/article-components/ArticleList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import {Link} from "react-router-dom";
+import * as api from "../../utils/api.js";
+import ArticleCard from "./ArticleCard.jsx";
+
+export default function ArticleList() {
+    const [articles, setArticles] = useState([]);
+    const[isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        setIsLoading(true);
+        api.getArticles().then((articles) => {
+            setArticles(articles)
+            setIsLoading(false)

```

Comment:
Error handling - .catch with an error state - look in the notes for a good guide.